frontends/web/src/routes/exchange/components/infocontent.tsx

```diff
@@ -139,14 +176,71 @@ export const BTCDirectInfo = ({ accounts }: TBTCDirectInfo) => {
       <p>{t('buy.exchange.infoContent.btcdirect.infobox.kyc')}</p>
       <br />
       <p>
-        <A href={getBTCDirectLink()}>
+        <A href={getBTCDirectOTCLink()}>
           {t('buy.exchange.infoContent.btcdirect.infobox.learnmore')}
         </A>
       </p>
     </div>
   );
 };
 
+type TBTCDirectInfoProps = {
+  cardFee?: number;
+  bankTransferFee?: number;
+  sofortFee?: number;
+  bancontactFee?: number;
+};
+
+const BTCDirectInfo = ({
+  cardFee,
+  bankTransferFee,
+  sofortFee,
+  bancontactFee,
+}: TBTCDirectInfoProps) => {
+  const { t } = useTranslation();
+  const formattedCardFee = cardFee && cardFee * 100;
+  const formattedBankTransferFee = bankTransferFee && bankTransferFee * 100;
+  const formattedSofortFee = sofortFee && sofortFee * 100;
+  const formattedBancontactFee = bancontactFee && bancontactFee * 100;
```

Comment:
this is not ideal, with 0.036 fee I get 3.5999999999999996%

probably better we do such math stuff in the backend 😇

Reply:
I think that we could store the fees in the backend deals as 3.6 instead of 0.036 so that we don't have to convert in the frontend. Would that be good enough?

Reply:
yes that makes sense. Looks to me that we do not use fee other than isBest calculation here https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/bbdc3b87016b33dc2b6ac46eb4a5cc0d65d2c1eb/backend/exchanges/exchanges.go#L214-L223

 change in last commit

Reply:
Awesome, ty

Reply:
changed to percentages in last commit.